index.js

```diff
@@ -2369,6 +2382,17 @@
 
   //. ### Object
 
+  //# prop :: Accessible a => String -> a -> b
```

Comment:
I wonder whether `String -> Record -> a` is a better signature. I certainly don't think we should mention Accessible unless we're actually using it.